packages/scene-composer/src/interfaces/components.ts

```diff
@@ -49,6 +51,7 @@ export interface IAnchorComponent extends ISceneComponent {
   offset?: Vector3;
   chosenColor?: string;
   customColors?: string[];
+  customIcon?: IconDefinition;
```

Comment:
So this will show up in our save data for the scene. Is there a way to abstract the type so we're not directly dependent on a 3rd party type in our save format?

Reply:
Yes. I will create an internal type as it makes sense to use our own type in case if they change their definition.

Reply:
Okay this is addressed.